spec/spec_helper.rb

```diff
@@ -60,6 +60,23 @@
     mocks.verify_partial_doubles = true
   end
 
+  # cribbed from https://gist.github.com/osulyanov/10609515 and https://rspec.info/documentation/3.13/rspec-core/RSpec/Core/Example.html
```

Comment:
Can you put in a comment that says what this is doing?

Reply:
yup, good call